charts/harness-delegate/v1.0.0/values.yaml

```diff
@@ -0,0 +1,72 @@
+# Default values for harness-delegate chart.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+# These are the required values that are needed for harness-delegate
+# chart deployment. You can download these account specific values by
+# going to Harness > Setup > Installations page
+
+# Account Id to which the delegate will be connecting
+accountId: ""
+# Account Secret
+accountSecret: ""
+# Short 6 character identifier of the account
+accountIdShort: ""
+
+
+# These are the prepopulated default values that are needed for
+# harness-delegate chart deployment
+
+# Delegate name
+delegateName: harness-delegate
+# If the RBAC scope is at cluster or namespace level
+clusterWideRbacScope: true
+
+# For connecting to Harness production SAAS environment, leave these
+# values as it is. For on-prem installation, these will be overriden
+# and you can download at Harness > Setup > Installations page
+managerHostAndPort: https://app.harness.io
+watcherStorageUrl: https://app.harness.io/storage/wingswatchers
+watcherCheckLocation: watcherprod.txt
+delegateStorageUrl: https://app.harness.io/storage/wingsdelegates
+delegateCheckLocation: delegateprod.txt
+
+# Delegate image to be deployed
+image: harness/delegate:latest
+
+# Delegate resource limits
+resources:
+  limits:
+    cpu: "1"
+    memory: "8Gi"
+  requests:
+    cpu: "1"
+    memory: "8Gi"
```

Comment:
can we set the `requests` resource to a little bit lower, otherwise user can't run this app in small k8s cluster.
e.g,
```YAML
  requests:
    cpu: "100m"
    memory: "200Mi"
```

Reply:
Updated it to below numbers and this is the minimum that we need for our delegate to come up:

requests:
    cpu: "600m"
    memory: "6Gi"